src/packages/emmett-mongodb/src/testing/mongoMocks.ts

```diff
@@ -0,0 +1,42 @@
+import type {
+  Collection,
+  CollectionOptions,
+  Db,
+  DbOptions,
+  Document,
+  MongoClient,
+} from 'mongodb';
+
+export const getDummyCollection = <TSchema extends Document = Document>(
```

Comment:
**COMMENT:** Intention behind this dummy collection is to be used in the structure tests, not to act as an in-memory collection.